src/fidesops/service/masking/strategy/masking_strategy.py

```diff
@@ -13,10 +13,20 @@ class MaskingStrategy(ABC):
     """Abstract base class for masking strategies"""
 
     @abstractmethod
-    def mask(self, value: Optional[str]) -> Optional[str]:
+    def mask(self, value: Optional[str], request_id: Optional[str]) -> Optional[str]:
         """Used to mask the provided value"""
         pass
 
+    @staticmethod
```

Comment:
Should this be a static method?

Reply:
Because it doesn't rely on instance vars, I usually prefer static methods.

Reply:
In the circumstances we're calling these methods we aren't calling them as staticmethods. If we're going to leave them static we should call them as such, it'll be confusing to developers otherwise. Consider removing the staticmethod annotation, since we're already instantiating the objects used to call the methods.

Reply:
ok makes sense!